xbout/plotting/utils.py

```diff
@@ -204,3 +204,65 @@ def _get_perp_vec(u1, u2, magnitude=0.04):
     v = np.linalg.norm((vx, vy))
     wx, wy = -vy / v * magnitude, vx / v * magnitude
     return wx, wy
+
+
+def _make_structured_triangulation(m, n):
+    # Construct a 'triangulation' of an (m x n) grid
+    # Returns an array of triples of (flattened) indices of the corners of the triangles
+    indices = np.zeros((m - 1, 2 * (n - 1), 3), dtype=np.uint32)
+    # indices[0] = [0, 1, n]
+    # indices[1] = [1, n+1, n]
+
+    indices[:, 0 : 2 * (n - 1) : 2, 0] = (
```

Comment:
It seems you first set the start points than the end points, but this is a bit confusing, maybe add an explanation what the indices are?

Also, I think this can be simplified:
```suggestion
    indices[:, 0 :: 2, 0] = (
```

Reply:
Or even
```suggestion
    indices[:, ::2, 0] = (
```